tests/unit/commands/deploy/test_command.py

```diff
@@ -39,6 +39,7 @@ def setUp(self):
         self.metadata = {}
         self.guided = False
         self.confirm_changeset = False
+        self.resolve_s3 = False
```

Comment:
Fine as a default, but can we add unit tests where this is set to true, and where we stub out the act of actually setting up the bucket?